src/transport_picoquic.cpp

```diff
@@ -215,9 +214,7 @@ pq_event_cb(picoquic_cnx_t* cnx,
         }
 
         default:
-            log_msg.str("");
-            log_msg << "Got event " << fin_or_event;
-            transport->logger.log(LogLevel::debug, log_msg.str());
+            LOGGER_DEBUG(transport->logger, "Got event " << fin_or_event);
```

Comment:
Should all the calls to `transport->logger->info` to `LOGGER_INFO()`?

Reply:
It depends on whether we want the ability to compile those out.  I just changed the debug logs since those are quite likely something we want to compile out.  The others we are more likely to want to make a run-time option.